pkg/engine/source/filesystem.go

```diff
@@ -365,6 +306,10 @@ func (s *FilesystemSource) iterateQueryDirs(queryDirs []string, queryParameters
 			continue
 		}
 
+		if !queryParameters.ExperimentalQueries && query.Experimental {
```

Comment:
```suggestion
		if query.Experimental && !queryParameters.ExperimentalQueries {
```

Since the majority of the queries do not have the `experimental` field, means the `query.Experimental` value is `false`, with the code suggestion, **we just check one of the if conditions** (because the first condition is `false `so the program do not need to check the second condition since it is a `&&` operator) for the majority of cases.

Reply:
Thanks @gabriel-cx